docs/api-docs/customers/customer-login-api.md

```diff
@@ -173,13 +171,12 @@ function getLoginUrl(customerId, storeHash, storeUrl, clientId, clientSecret) {
    return `${storeUrl}/login/token/${token}`;
 };
  
-const clientId = “Your client id”;
 const clientSecret = “Your client secret”;
 const customerId = “You customer id”;
 const storeHash = “Your store hash”;
 const storeUrl = “Your store url”;
  
-const loginUrl = getLoginUrl(customerId, storeHash, storeUrl, clientId, clientSecret);
+const loginUrl = getLoginUrl(customerId, storeHash, storeUrl, clientSecret);
```

Comment:
We can't just remove the clientId param here; we might need to re-write all the example code.